android/app/build.gradle

```diff
@@ -44,7 +45,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "org.helpapaw.helpapaw"
```

Comment:
Why did you change the app Id?

Reply:
firebase config will create with 'com' that's y I changed the application id.

Reply:
I changed the configuration to use the original app Ids. So, it stays the same after firebase config.